QUARKUS-1070.md

```diff
@@ -0,0 +1,39 @@
+# QUARKUS-1070 Composable Platform streams and non-platform extension discovery
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1070
+
+The goal of this epic is to
+
+* introduce a registry.quarkus.redhat.com for all devtools to consult for looking up platform and extensions
+* introduce bom generator in platform that generates uniform but composable BOMs that can be imported individually 
+  rather than all in bulk 
+* introduce notion of platform stream to indicate which minor version you are tracking and/or what you want to target 
+  when using tools
+
+## Scope of the testing
+
```

Comment:
Pls link "Starting point document for QE participants" https://docs.google.com/document/d/1qCOu4RiONV2PTTitJzJvRdnXYbUX2au5SXeF3ApupCI/edit
And mention that this is the initial plan from our side and that things are evolving ...

Reply:
This is public repo, and we probably shouldn't show that link even in comments

Reply:
It's private document, so I don't think it's a big deal to have it even in comments